$NetBSD: patch-CMakeLists.txt,v 1.3 2025/04/03 09:21:45 wiz Exp $

Only c++17 is needed when WITH_UNCOMPRESSED_CODEC is OFF (default as of 1.19.7).
https://github.com/strukturag/libheif/issues/1500

--- CMakeLists.txt.orig	2025-03-03 11:00:24.000000000 +0000
+++ CMakeLists.txt
@@ -48,7 +48,7 @@ if(NOT MSVC)
   endif ()
 endif()
 
-set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
